src/components/Tabs/Annotation/AnnotationTab.tsx

```diff
@@ -175,16 +169,6 @@ export default function AnnotationTab(props: AnnotationTabProps): ReactElement {
           disabled={currentLabelIdx === null}
           showSelectedItemTooltip={false}
         ></SelectionDropdown>
-        <div>
-          {/* TODO: Remove color picker once color dots can be added to the dropdowns. */}
-          <ColorPicker
-            size="small"
-            value={new AntdColor(selectedLabel?.color.getHexString() || "#ffffff")}
-            onChange={onColorPickerChange}
-            disabledAlpha={true}
-            disabled={!isAnnotationModeEnabled}
-          />
-        </div>
```

Comment:
it seems like the user no longer as the ability to assign or change a color? will that be added back?

Reply:
You have to activate the edit mode and then click the edit button to open a popup.